File: ma331/hw2/hw2.py
from statistics import *
from datetime import datetime
import csv
import math

#for hw2 because its stupid and i need data

data = {}

n = 100
s_mean = 2
std = 9

def getSample(i):
    sT = datetime.now()
    return NormalDist(s_mean, std).samples(n, seed = i)



with open('hw2_100.csv', mode = '+w') as csv_file:
    fieldnames = ['equation1', 'equation2']
    writer = csv.DictWriter(csv_file, fieldnames=fieldnames)

    writer.writeheader()
    for i in range(100):
        y = getSample(i)
        eq1 = (mean(y) - 2)/(math.sqrt(std/n))
        eq2 = ((n-1)*stdev(y)) / std
        writer.writerow({'equation1': eq1, 'equation2' : eq2})

File: ma331/hw2/hw2.r
n <<- 20
eq1 <- c()
eq2 <- c()
data1 <<- data.frame(eq1,eq2)
for (I in 0:99){
  d <<- rnorm(n, 2, 9)
  equ1 <- (mean(d)-2)/(sqrt(9/n))
  equ2 <- ((n-1)* sd(d))/9
  data1.df <- rbind(I,c(equ1, equ2))
}

